### PR TITLE
Fix the class link and overview for Events

### PR DESCRIPTION
**Pull request by adbridge**

### CODE REVIEW

**Comment by adbridge** [comment]
@donatieng are you able to test this PR to make sure it renders correctly ?

File: docs/api/scheduling/Event.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Event
 
-The Event class provides APIs to configure events `delay` and `period` timings. You can use `post` API to post an event to the underlying EventQueue, and you can use `cancel` to cancel the most recently posted event.
+The Event class provides APIs to configure events `delay` and `period` timings. The timings use the std::chrono::duration class provided by the <utility> library. You can use `post` API to post an event to the underlying EventQueue, and you can use `cancel` to cancel the most recently posted event.
```
Comment (donatieng) [inline docs/api/scheduling/Event.md]:
```suggestion
The Event class provides APIs to configure events `delay` and `period` timings. The timings use the `std::chrono::duration` class provided by the `<utility>` library. You can use `post` API to post an event to the underlying EventQueue, and you can use `cancel` to cancel the most recently posted event.
```

**Review by donatieng — CHANGES_REQUESTED** [review]
Thanks @adbridge looks good, however `<utility>` seems to be eaten by the docs engine: https://os.mbed.com/docs/mbed-os/event-fix/apis/event-handling-apis.html

File: docs/api/scheduling/Event.md (hunk 1 above)
Comment (adbridge) [reply docs/api/scheduling/Event.md]: Interesting! I'll add the quotes!

**Comment by adbridge** [comment]
@donatieng can you try again now?

**Comment by donatieng** [comment]
Waiting for CI :)

**Review by donatieng — APPROVED** [review]
Thanks @adbridge, seems to work now :)

**Pull request merged**